platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/collections/LongListDisk.java

```diff
@@ -150,10 +313,14 @@ protected void writeLongsData(final FileChannel fc) {}
     @Override
     protected long lookupInChunk(final long chunkIndex, final long subIndex) {
         try {
-            final ByteBuffer buf = TEMP_LONG_BUFFER_THREAD_LOCAL.get();
-            final long offset = FILE_HEADER_SIZE + (((chunkIndex * numLongsPerChunk) + subIndex) * Long.BYTES);
+            final long listIndex = (chunkIndex * numLongsPerChunk) + subIndex;
+            if (listIndex < minValidIndex.get()) {
+                return IMPERMISSIBLE_VALUE;
+            }
+            final ByteBuffer buf = tempLongBufferThreadLocal.get();
+            final long offset = createOrGetChunk(listIndex) + calculateOffsetInChunk(listIndex);
```

Comment:
`listIndex` or `chunkIndex`?

Reply:
`listIndex`.

but this `calculateOffsetInChunk(listIndex)`  can be replaced with `subIndex * Long.BYTES`, if it makes it more readable.